client/protocols/unlock.py

```diff
@@ -150,6 +155,18 @@ def interact(self, unique_id, case_id, question_prompt, answer, choices=None, ra
 
             else:
                 correct = True
+            tg_id = -1
+            misU_count = {}
```

Comment:
I'd prefer something like `misU_counts`, but a comment with what you just told me would probably be more helpful.